### ISSUE TITLE
Milestone4 License discussion

### ISSUE THREAD

**Issue by nobbynguyen** [UBC-MDS/CryptocurrencyEDA#71]
- Examine the license for our project and consider whether this is the choice we want to make, or whether we want to change the license.
- Discuss and reason the license choice.

**Comment by ciciecho-ds** [comment]
I think we can continue using the MIT license because it has the "legal mileage" that Tiffany mentioned in the recording, which means that people are familiar with it and may feel more comfortable using our package or build upon it when they see our choice of license. I also took a look at this site: https://choosealicense.com/licenses/mit/ and I think the permissions, conditions and limitations listed suit our needs. MIT license is also one of the most popular licenses for work distributed both on PyPI and CRAN.

**Comment by nobbynguyen** [comment]
Thank you Cici for sharing your thought. I also think MIT license is a good choice because of the following reasons: 
- MIT license is in common use, so the contributors and users are more familiar with it. As we are willing to make this package free and open to public, a common use of license will encourage people using our package. 
- MIT is the very permissive. It’s not only friendly to open source developers but also to businesses. Using MIT license would provide us more flexibility in case we would want future improvement.

**Comment by berkaybulut** [comment]
Thank you Cici for point it out! I agree with Nobby MIT license is a good choice because it is in common use, a lot of packages use MIT and developers and users are used to it. Secondly it allows for open source developers to develop on top and allows businesses to utilize it as well. I believe using MIT is a good way to go.

**Comment by alexYinanGu0** [comment]
I agree with the idea of using MIT as well! I think that MIT license is appropriate for our project. We certainly do not want any restriction on the distribution. We would like our project to be a collaborative effort. Our project should be open to feedback and improvements.
